index.html

```diff
@@ -915,12 +915,182 @@ <h4>Get a Specific Presentation</h4>
+is compatible with how the client received the exchange URL. For example, the
+exchange URL could have been provided over CHAPI with a protocol identifier
+indicating that the VC API protocol ought to be used. Alternatively, the
+exchange URL could be included as the "credential_issuer" in an OID4VCI
+credential offer, or as the "client_id" of an OID4VP authorization request,
+indicating that OID4VCI or OID4VP, respectively, ought to be used. This section
+focuses on how an exchange client uses VC API to interact with the exchange,
+but see appendix X to see how to combine VC API exchanges with other protocols
+such as OID4VCI, OID4VP, and DIDComm.
+      </p>
+      <p>
+Exchanges that are executed using the VC API protocol involve messages sent as
+request and response bodies over HTTP. Each message consists of a simple JSON
+object that includes zero or more of the following properties and values:
+      </p>
+      <ul class="bullet">
+        <li>
+`verifiablePresentationRequest`: A Verifiable Presentation Request. This is
+used by one party in the exchange to request information from the other party.
+        </li>
+        <li>
+`verifiablePresentation`: A Verifiable Presentation. This is used by one party
+to provide information requested by the other party or simply offered by the
+sender.
+        </li>
+        <li>
+`redirectUrl`: A URL that can be used to continue an interaction at another
+location. One use case for this is to redirect the user of an exchange client
+back to a coordinator website after an exchange has completed.
+        </li>
+        <li>
+Any custom properties and values might also be included, but are expected to
+trigger errors in implementations if they are not recognized.
+        </li>
+      </ul>
+      <p>
+To initiate the exchange using the VC API protocol, the exchange client
+performs an HTTP POST sending a JSON object as the request body. In the
+simplest case, the JSON object is empty (`{}`), for the case when the client
+has no constraints of its own on the exchange, i.e., it has nothing to request
+from the other party. The workflow service responds with its own JSON object
+in the response body.
+      </p>
+      <p>
+If that object is empty, the exchange is complete and nothing is requested nor
+offered to the exchange client. If the object includes
+`verifiablePresentationRequest`, then the exchange is not yet complete and
+some additional information is requested, based on the contents of the
+associated verifiable presentation request. If the object includes
+`verifiablePresentation`, then some information is offered, such as issued
+verifiable credentials or verifiable credentials provided in response to the
+exchange client's request. If the object includes `redirectUrl`, the exchange
+is complete and the workflow service recommends that the client proceed to
+another place to continue the interaction in another form.
```

Comment:
This is a partial rewording. More is needed, especially on the `verifiablePresentation` case. It's not clear to me what is meant, so I cannot rephrase that sentence. The troublesome bit is `such as issued verifiable credentials or verifiable credentials provided in response to the exchange client's request` where it's not clear to me what the alternatives are. Maybe `such as previously issued verifiable credentials or newly issued verifiable credentials provided in response to the exchange client's request` would work here? (It would likely help if things were marked up with `[=these=]` and related wrappers. Why aren't those liberally sprinkled over this text?)

```suggestion
If that response object is empty, the exchange is complete and nothing is
requested from nor offered to the exchange client. If the object includes
`verifiablePresentationRequest`, then the exchange is not yet complete and
some additional information is requested, as specified by the contents of the
associated verifiable presentation request. If the object includes
`verifiablePresentation`, then some information is offered, such as issued
verifiable credentials or verifiable credentials provided in response to the
exchange client's request. If the object includes `redirectUrl`, the exchange
is complete and the workflow service recommends that the client proceed to
another place to continue the interaction in another form.
```

Reply:
Adding suggestion and will come back to this to resolve the rest of the comment.

Reply:
Ok, I've tried to clarify this with the latest changes. In short, a VP could include VCs that were issued to the holder that is operating the exchange client OR the VCs could be about whomever is operating the exchange server, i.e., if the holder used their exchange client client to ask a website to provide its own credentials before the holder is willing to continue in the exchange.

Reply:
Regarding `[=foo=]` ... that pattern doesn't exist at all in the spec at this time (AFAICT). A subsequent PR that adds that sort of thing will need to be done over the whole document, I expect.